src/sonic-db-cli/sonic-db-cli.cpp

```diff
@@ -0,0 +1,239 @@
+    std::cout << "Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu" << std::endl;
+    std::cout << "Example 4: sonic-db-cli -n asic3 APPL_DB EVAL \"return {KEYS[1],KEYS[2],ARGV[1],ARGV[2]}\" 2 k1 k2 v1 v2" << std::endl;
+    std::cout << "Example 5: sonic-db-cli PING | sonic-db-cli -s PING" << std::endl;
+    std::cout << "Example 6: sonic-db-cli SAVE | sonic-db-cli -s SAVE" << std::endl;
+    std::cout << "Example 7: sonic-db-cli FLUSHALL | sonic-db-cli -s FLUSHALL" << std::endl;
+}
+
+void printRedisReply(redisReply* reply)
+{
+    switch(reply->type)
+    {
+    case REDIS_REPLY_INTEGER:
+        std::cout << reply->integer;
+        break;
+    case REDIS_REPLY_STRING:
+    case REDIS_REPLY_ERROR:
+    case REDIS_REPLY_STATUS:
+    case REDIS_REPLY_NIL:
+        std::cout << std::string(reply->str, reply->len);
+        break;
+    case REDIS_REPLY_ARRAY: 
+        for (size_t i = 0; i < reply->elements; i++)
+        {
+            printRedisReply(reply->element[i]);
+        }
+        break;
+    default:
+        std::cerr << reply->type << std::endl;
+        throw std::runtime_error("Unexpected reply type");
+    }
+
+    std::cout << std::endl;
+}
+
+int executeCommands(
+    const std::string& db_name,
+    std::vector<std::string>& commands,
+    const std::string& name_space,
+    bool use_unix_socket)
+{
+    std::unique_ptr<swss::SonicV2Connector_Native> dbconn;
+    if (name_space.compare("None") == 0) {
+        dbconn = std::make_unique<swss::SonicV2Connector_Native>(use_unix_socket, empty_str);
+    }
+    else {
+        dbconn = std::make_unique<swss::SonicV2Connector_Native>(true, name_space.c_str());
+    }
+
+    try {
+        dbconn->connect(db_name);
+    }
+    catch (const std::exception& e)
+    {
+        std::cerr << "Invalid database name input : " << db_name << std::endl;
+        return 1;
+    }
+
+    auto& client = dbconn->get_redis_client(db_name);
+
+    size_t argc = commands.size();
+    const char** argv = new const char*[argc];
+    size_t* argvc = new size_t[argc];
+    for (size_t i = 0; i < argc; i++)
+    {
+        argv[i] = strdup(commands[i].c_str());
+        argvc[i] = commands[i].size();
+    }
+
+    swss::RedisCommand command;
+    command.formatArgv(argc, argv, argvc);
+    swss::RedisReply reply(&client, command);
+
+    auto redisReply = reply.getContext();
+    printRedisReply(redisReply);
+
+    return 0;
+}
+
+void handleSingleOperation(
+    const std::string& name_space,
+    const std::string& db_name,
+    const std::string& operation,
+    bool use_unix_socket)
+{
+    swss::SonicV2Connector_Native conn(use_unix_socket, name_space.c_str());
+    conn.connect(db_name);
+    auto& client = conn.get_redis_client(db_name);
+    
+    swss::RedisReply reply(&client, operation);
+    auto redisReply = reply.getContext();
+    printRedisReply(redisReply);
+}
+
+void handleAllInstances(
+    const std::string& name_space,
```

Comment:
>name_space [](http://example.com/codeflow?start=23&length=10)

We name it like `ns_name` or `m_namespace` as in other c++ code in sonic-swss-common repo.

Reply:
Fixed, will submit in another PR, because this file will check-in to sonic-swss-common repo.